### PR TITLE
misc: move to std::print

### PR DESCRIPTION
**Pull request by PaideiaDilemma**

### CODE REVIEW

**Review by vaxerski — APPROVED** [review]

**Comment by vaxerski** [comment]
conflict tho

**Comment by littleblack111** [comment]
wouldn't it be better if u replace cerr to print(stderr, ...)

**Comment by PaideiaDilemma** [comment]
@littleblack111 
I think help should not be printed to stderr.
But I can change it back. Do not have a strong opinion on it.

**Review by littleblack111 — CHANGES_REQUESTED** [review]

File: src/main.cpp [hunk 1]
```diff
@@ -17,14 +17,14 @@ void help() {
...
                  "  -V, --version            - Show version information\n"
-                 "  -h, --help               - Show this help message\n";
+                 "  -h, --help               - Show this help message");
 }
 
 std::optional<std::string> parseArg(const std::vector<std::string>& args, const std::string& flag, std::size_t& i) {
     if (i + 1 < args.size()) {
         return args[++i];
     } else {
-        std::cerr << "Error: Missing value for " << flag << " option.\n";
```
[the remark this answers is not in the document]
Comment (PaideiaDilemma) [reply src/main.cpp:27]: ups yeah thanks!

**Review by littleblack111 — APPROVED** [review]

**Pull request merged**